### PR TITLE
fixes linter warnings on EditProfile.js component

### PR DESCRIPTION
**Pull request by justinpeisker**
## Description

Super small PR to fix warnings when login page loads.  Not a huge deal but its been bugging me for like a week 😂

#### Video Link

https://www.loom.com/share/af9b35efaa314c3da2341c9da84312e1

#### Trello Link

https://trello.com/c/tqZjVMzK

## Type of change

___Please delete options that are not relevant.___

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by prashishb — APPROVED** [review]
Nice job cleaning up those linter warnings. Makes code clean and more pleasant. GJ!

**Review by robstetson — APPROVED** [review]
Small, but needed PR. Great Job catching those linter warnings and getting them taken care of quickly.

**Review by yefim-g — APPROVED** [review]
Always good to fix any linter warnings!

**Review by maidiance — CHANGES_REQUESTED** [review]
Thank you for taking care of these linter warnings. Do you also see the Github Lint on Line 21 for the component in src/components/pages/Notes/Note.js? I don't know what's causing it just from the snippet I can see in Github. Maybe you can take a look?

**Review by adamhinton — APPROVED** [review]
I love to see this. Small, modular PRs are the foundation of good web dev.

**Comment by justinpeisker** [comment]
> Thank you for taking care of these linter warnings. Do you also see the Github Lint on Line 21 for the component in src/components/pages/Notes/Note.js? I don't know what's causing it just from the snippet I can see in Github. Maybe you can take a look?

So I did some digging and the warnings are because a few divs needed unique keys.  I'll have to submit a different PR since it is in a file that is not associated with this PR. (At least as far as I can tell I'm not able to edit it.  But could be wrong)

**Comment by maidiance** [comment]
> > Thank you for taking care of these linter warnings. Do you also see the Github Lint on Line 21 for the component in src/components/pages/Notes/Note.js? I don't know what's causing it just from the snippet I can see in Github. Maybe you can take a look?
> 
> So I did some digging and the warnings are because a few divs needed unique keys. I'll have to submit a different PR since it is in a file that is not associated with this PR. (At least as far as I can tell I'm not able to edit it. But could be wrong)

Thank you for checking this, awesome work 👍

**Review by acoman-cloud — APPROVED** [review]

File: src/components/pages/Profile/EditProfile.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
-import React, { useState, useEffect } from 'react';
```
Comment (phillipgalbraith) [inline src/components/pages/Profile/EditProfile.js:1]: Cleaning up the code this way can help save storage and make the code easier to read.  Thanks!

**Review by phillipgalbraith — COMMENTED** [review]
This makes me wonder why useEffect was in the file in the first place.  I wonder if the purpose of the EditProfile has changed so taht inneeds

**Review by itsmerobert12 — APPROVED** [review]
Crazy how the smallest things cause the biggest itches!!! Nice PR fix here. I would recommend looking into the minor warnings but I'm pretty sure that's a small fix.

**Review by BryceSlade — APPROVED** [review]
Thank you for this! Let's get rid of these warnings once and for all!

**Review by areccus — APPROVED** [review]

**Review by jonathan-mcgraw — APPROVED** [review]
Beautiful, straight forward small fix that improves functionality. I love it!

File: src/components/pages/Profile/EditProfile.js (hunk 1 above)
Comment (morgankj) [reply src/components/pages/Profile/EditProfile.js:1]: Errors like this are the bane of my existence. Thank you for removing this!

**Review by morgankj — APPROVED** [review]
Great job! Small changes make code much more livable. Thank you!

**Review by hollberg — APPROVED** [review]
Removing unused imports makes for safer code (smaller attack surface). And leaving even 'minor' linter errors in the code base desensitizes devs from attending to potentially bigger issues. Very developer friendly fix!

**Review by jcpcab — APPROVED** [review]
Thank you for working on this on finally taking out these warnings!

**Pull request merged**